src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -351,6 +351,46 @@ export default class ClinicalData extends React.Component<IClinicalDataProps, {}
         }
     }
 
+    @computed get boxPlotTooltip() {
+        return (d: any) => {
+            let content;
+            if (this.boxPlotData.isComplete) {
+                content = <table className="table table-bordered table-striped">
```

Comment:
would rather not have the table-bordered unless super necessary.  just for consistency

Reply:
done